webui/vendor/Bareos/library/Bareos/BSock/BareosBSock.php

```diff
@@ -964,7 +964,7 @@ public function restore($jobid=null, $client=null, $restoreclient=null, $restore
          $debug = self::receive_message();
       }
 
-      if(self::send('restore file=?b2000'.$rnd.' client='.$client.' restoreclient='.$restoreclient.' restorejob="'.$restorejob.'" where='.$where.' replace='.$replace.' yes')) {
+      if(self::send('restore file=?b2000'.$rnd.' client="'.$client.'" restoreclient="'.$restoreclient.'" restorejob="'.$restorejob.'" where="'.$where.'" replace='.$replace.' yes')) {
```

Comment:
doesn't it make sense to also quote the `replace` option?

Reply:
Well, you can argue here. The current possible replace values are "always, never, ifnewer and ifolder". But it's propably a good idea to also quote that argument value, just in case sth. changes in future.

Reply:
Suggestion adapted.